txpipe/twopoint.py

```diff
@@ -283,6 +288,7 @@ def add_data_points(self, S, results):
                         error=xiperr[i],
                         npair=npair[i],
                         weight=weight[i],
+                        tracers_later=tracers_later,
```

Comment:
what does this thing do? (if answered later disregard my message)

Reply:
One of the tracers is the external map. This isn't a known tracer in sacc so it gives an error unless you set tracers_later=True